[2025-03-19T11:01:46.532+0000] {processor.py:186} INFO - Started process (PID=66) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:01:46.534+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:01:46.541+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:01:46.541+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:01:46.583+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:01:46.582+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:01:46.584+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:01:47.050+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.569 seconds
[2025-03-19T11:03:16.421+0000] {processor.py:186} INFO - Started process (PID=142) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:16.447+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:03:16.451+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:03:16.450+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:16.521+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:03:16.454+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:03:16.522+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:16.785+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.442 seconds
[2025-03-19T11:03:47.068+0000] {processor.py:186} INFO - Started process (PID=204) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:47.072+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:03:47.076+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:03:47.075+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:47.095+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:03:47.089+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:03:47.096+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:03:47.148+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.098 seconds
[2025-03-19T11:04:19.070+0000] {processor.py:186} INFO - Started process (PID=266) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:04:19.078+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:04:19.082+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:04:19.082+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:04:19.095+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:04:19.086+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:04:19.095+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:04:19.157+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.096 seconds
[2025-03-19T11:07:39.041+0000] {processor.py:186} INFO - Started process (PID=296) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:07:39.058+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:07:39.060+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:07:39.059+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:07:39.119+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:07:39.077+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:07:39.120+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:07:39.487+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.473 seconds
[2025-03-19T11:08:56.683+0000] {processor.py:186} INFO - Started process (PID=363) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:08:56.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:08:56.701+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:08:56.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:08:56.718+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:08:56.710+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:08:56.719+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:08:56.771+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.098 seconds
[2025-03-19T11:09:27.410+0000] {processor.py:186} INFO - Started process (PID=423) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:27.415+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:09:27.417+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:09:27.416+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:27.435+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:09:27.432+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:09:27.436+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:27.479+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.082 seconds
[2025-03-19T11:09:58.484+0000] {processor.py:186} INFO - Started process (PID=487) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:58.487+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:09:58.490+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:09:58.489+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:58.509+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:09:58.497+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:09:58.510+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:09:58.558+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.082 seconds
[2025-03-19T11:10:30.809+0000] {processor.py:186} INFO - Started process (PID=543) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:10:30.816+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:10:30.818+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:10:30.818+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:10:30.833+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:10:30.827+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:10:30.834+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:10:30.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.106 seconds
[2025-03-19T11:11:07.287+0000] {processor.py:186} INFO - Started process (PID=606) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:11:07.304+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:11:07.308+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:11:07.307+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:11:07.328+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:11:07.312+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:11:07.328+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:11:07.378+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.144 seconds
[2025-03-19T11:12:19.636+0000] {processor.py:186} INFO - Started process (PID=643) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:12:19.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:12:19.654+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:12:19.653+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:12:19.742+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:12:19.693+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:12:19.743+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:12:20.659+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 1.105 seconds
[2025-03-19T11:15:48.222+0000] {processor.py:186} INFO - Started process (PID=704) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:15:48.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:15:48.235+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:15:48.231+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:15:48.388+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:15:48.242+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:15:48.389+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:15:48.605+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.409 seconds
[2025-03-19T11:17:05.646+0000] {processor.py:186} INFO - Started process (PID=732) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:17:05.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:17:05.691+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:17:05.690+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:17:05.748+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:17:05.697+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:17:05.755+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:17:05.954+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.364 seconds
[2025-03-19T11:28:13.839+0000] {processor.py:186} INFO - Started process (PID=787) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:28:13.842+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T11:28:13.844+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:28:13.843+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:28:13.852+0000] {logging_mixin.py:190} INFO - [2025-03-19T11:28:13.849+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 57
    ) as dag:
             ^
IndentationError: expected an indented block after 'with' statement on line 51
[2025-03-19T11:28:13.853+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T11:28:13.890+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.065 seconds
[2025-03-19T22:41:23.814+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:23.817+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:41:23.823+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:41:23.822+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:23.838+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:41:23.836+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:41:23.839+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:23.903+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.107 seconds
[2025-03-19T22:41:56.123+0000] {processor.py:186} INFO - Started process (PID=131) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:56.134+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:41:56.138+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:41:56.138+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:56.152+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:41:56.143+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:41:56.154+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:41:56.235+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.129 seconds
[2025-03-19T22:42:27.452+0000] {processor.py:186} INFO - Started process (PID=189) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:27.455+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:42:27.460+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:42:27.459+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:27.464+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:42:27.463+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:42:27.464+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:27.520+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.084 seconds
[2025-03-19T22:42:57.909+0000] {processor.py:186} INFO - Started process (PID=253) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:57.915+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:42:57.919+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:42:57.918+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:57.926+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:42:57.924+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:42:57.927+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:42:57.981+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.082 seconds
[2025-03-19T22:43:28.655+0000] {processor.py:186} INFO - Started process (PID=315) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:28.677+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:43:28.682+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:43:28.681+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:28.699+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:43:28.696+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:43:28.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:28.914+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.294 seconds
[2025-03-19T22:43:59.499+0000] {processor.py:186} INFO - Started process (PID=377) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:59.502+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:43:59.505+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:43:59.505+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:59.521+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:43:59.518+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:43:59.522+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:43:59.573+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.101 seconds
[2025-03-19T22:44:30.204+0000] {processor.py:186} INFO - Started process (PID=440) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:44:30.206+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:44:30.209+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:44:30.209+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:44:30.227+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:44:30.225+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:44:30.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:44:30.325+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.140 seconds
[2025-03-19T22:45:00.472+0000] {processor.py:186} INFO - Started process (PID=504) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:00.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:45:00.477+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:45:00.477+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:00.483+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:45:00.481+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:45:00.483+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:00.510+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.049 seconds
[2025-03-19T22:45:30.685+0000] {processor.py:186} INFO - Started process (PID=568) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:30.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:45:30.700+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:45:30.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:30.715+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:45:30.705+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:45:30.716+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:45:30.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.133 seconds
[2025-03-19T22:46:00.892+0000] {processor.py:186} INFO - Started process (PID=631) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:00.893+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:46:00.897+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:46:00.895+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:00.903+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:46:00.902+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:46:00.904+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:00.933+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T22:46:31.575+0000] {processor.py:186} INFO - Started process (PID=693) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:31.591+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:46:31.595+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:46:31.594+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:31.604+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:46:31.602+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:46:31.604+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:46:31.641+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.073 seconds
[2025-03-19T22:47:02.155+0000] {processor.py:186} INFO - Started process (PID=758) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:02.156+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:47:02.161+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:47:02.160+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:02.167+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:47:02.166+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:47:02.168+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:02.222+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.086 seconds
[2025-03-19T22:47:32.367+0000] {processor.py:186} INFO - Started process (PID=822) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:32.369+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:47:32.372+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:47:32.372+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:32.383+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:47:32.381+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:47:32.383+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:47:32.431+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.078 seconds
[2025-03-19T22:48:06.437+0000] {processor.py:186} INFO - Started process (PID=877) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:06.527+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:48:06.531+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:48:06.530+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:06.551+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:48:06.538+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:48:06.552+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:07.351+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.947 seconds
[2025-03-19T22:48:39.913+0000] {processor.py:186} INFO - Started process (PID=930) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:39.940+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:48:39.945+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:48:39.944+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:39.970+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:48:39.968+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:48:39.971+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:48:40.249+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.361 seconds
[2025-03-19T22:49:10.718+0000] {processor.py:186} INFO - Started process (PID=992) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:10.721+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:49:10.725+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:49:10.725+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:10.742+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:49:10.740+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:49:10.743+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:10.851+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.157 seconds
[2025-03-19T22:49:41.324+0000] {processor.py:186} INFO - Started process (PID=1055) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:41.328+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:49:41.341+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:49:41.341+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:41.359+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:49:41.357+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:49:41.372+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:49:41.526+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.234 seconds
[2025-03-19T22:50:11.933+0000] {processor.py:186} INFO - Started process (PID=1119) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:11.935+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:50:11.939+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:50:11.938+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:11.955+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:50:11.953+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:50:11.956+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:12.027+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.109 seconds
[2025-03-19T22:50:42.354+0000] {processor.py:186} INFO - Started process (PID=1182) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:42.356+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:50:42.363+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:50:42.361+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:42.370+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:50:42.368+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:50:42.371+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:50:42.408+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.069 seconds
[2025-03-19T22:51:12.626+0000] {processor.py:186} INFO - Started process (PID=1246) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:12.642+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:51:12.646+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:51:12.645+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:12.660+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:51:12.658+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:51:12.661+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:12.713+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.111 seconds
[2025-03-19T22:51:43.336+0000] {processor.py:186} INFO - Started process (PID=1309) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:43.338+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:51:43.341+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:51:43.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:43.349+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:51:43.347+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:51:43.349+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:51:43.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.067 seconds
[2025-03-19T22:52:13.510+0000] {processor.py:186} INFO - Started process (PID=1373) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:13.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:52:13.518+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:52:13.518+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:13.525+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:52:13.523+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:52:13.526+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:13.555+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.064 seconds
[2025-03-19T22:52:43.730+0000] {processor.py:186} INFO - Started process (PID=1437) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:43.735+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:52:43.756+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:52:43.756+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:43.762+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:52:43.761+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:52:43.763+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:52:43.802+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.084 seconds
[2025-03-19T22:53:14.650+0000] {processor.py:186} INFO - Started process (PID=1501) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:14.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:53:14.695+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:53:14.673+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:14.712+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:53:14.710+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:53:14.744+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:14.885+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.267 seconds
[2025-03-19T22:53:45.112+0000] {processor.py:186} INFO - Started process (PID=1564) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:45.115+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:53:45.125+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:53:45.119+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:45.132+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:53:45.130+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:53:45.133+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:53:45.238+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.143 seconds
[2025-03-19T22:54:15.499+0000] {processor.py:186} INFO - Started process (PID=1625) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:15.504+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:54:15.508+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:54:15.508+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:15.516+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:54:15.514+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:54:15.518+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:15.629+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.151 seconds
[2025-03-19T22:54:45.876+0000] {processor.py:186} INFO - Started process (PID=1689) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:45.878+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:54:45.882+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:54:45.881+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:45.887+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:54:45.886+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:54:45.888+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:54:45.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.061 seconds
[2025-03-19T22:55:20.791+0000] {processor.py:186} INFO - Started process (PID=1753) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:20.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:55:20.826+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:55:20.823+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:20.839+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:55:20.837+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:55:20.840+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:20.899+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.154 seconds
[2025-03-19T22:55:51.179+0000] {processor.py:186} INFO - Started process (PID=1815) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:51.182+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:55:51.185+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:55:51.185+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:51.190+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:55:51.189+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:55:51.191+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:55:51.220+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.054 seconds
[2025-03-19T22:56:21.447+0000] {processor.py:186} INFO - Started process (PID=1874) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:21.449+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:56:21.454+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:56:21.453+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:21.467+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:56:21.464+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:56:21.467+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:21.500+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.070 seconds
[2025-03-19T22:56:53.434+0000] {processor.py:186} INFO - Started process (PID=1933) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:53.456+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:56:53.483+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:56:53.471+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:53.618+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:56:53.616+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:56:53.649+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:56:53.799+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.411 seconds
[2025-03-19T22:57:27.502+0000] {processor.py:186} INFO - Started process (PID=1986) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:27.504+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:57:27.511+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:57:27.510+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:27.517+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:57:27.516+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:57:27.518+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:27.573+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.095 seconds
[2025-03-19T22:57:59.204+0000] {processor.py:186} INFO - Started process (PID=2035) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:59.209+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:57:59.213+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:57:59.213+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:59.250+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:57:59.248+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:57:59.251+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:57:59.504+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.345 seconds
[2025-03-19T22:58:29.913+0000] {processor.py:186} INFO - Started process (PID=2098) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:58:29.916+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:58:29.921+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:58:29.920+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:58:29.941+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:58:29.939+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:58:29.943+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:58:30.050+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.160 seconds
[2025-03-19T22:59:00.812+0000] {processor.py:186} INFO - Started process (PID=2162) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:00.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:59:00.830+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:59:00.830+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:00.837+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:59:00.835+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:59:00.838+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:00.869+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.074 seconds
[2025-03-19T22:59:49.819+0000] {processor.py:186} INFO - Started process (PID=2229) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:49.917+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T22:59:49.920+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:59:49.920+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:49.924+0000] {logging_mixin.py:190} INFO - [2025-03-19T22:59:49.923+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T22:59:49.924+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T22:59:51.859+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 2.050 seconds
[2025-03-19T23:00:26.929+0000] {processor.py:186} INFO - Started process (PID=2281) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:26.940+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:00:26.944+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:00:26.943+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:26.999+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:00:26.997+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:00:27.006+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:27.116+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.198 seconds
[2025-03-19T23:00:58.045+0000] {processor.py:186} INFO - Started process (PID=2344) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:58.051+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:00:58.057+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:00:58.057+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:58.074+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:00:58.071+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:00:58.075+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:00:58.136+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.114 seconds
[2025-03-19T23:01:28.383+0000] {processor.py:186} INFO - Started process (PID=2407) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:28.395+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:01:28.399+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:01:28.398+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:28.407+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:01:28.404+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:01:28.408+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:28.471+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.109 seconds
[2025-03-19T23:01:58.777+0000] {processor.py:186} INFO - Started process (PID=2471) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:58.779+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:01:58.781+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:01:58.781+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:58.786+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:01:58.784+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:01:58.786+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:01:58.810+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.041 seconds
[2025-03-19T23:02:29.108+0000] {processor.py:186} INFO - Started process (PID=2535) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:29.110+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:02:29.112+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:02:29.112+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:29.118+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:02:29.117+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:02:29.119+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:29.151+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.054 seconds
[2025-03-19T23:02:59.320+0000] {processor.py:186} INFO - Started process (PID=2600) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:59.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:02:59.326+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:02:59.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:59.333+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:02:59.330+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:02:59.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:02:59.364+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.058 seconds
[2025-03-19T23:03:30.055+0000] {processor.py:186} INFO - Started process (PID=2662) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:03:30.056+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:03:30.065+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:03:30.064+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:03:30.071+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:03:30.069+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:03:30.071+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:03:30.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.069 seconds
[2025-03-19T23:04:00.730+0000] {processor.py:186} INFO - Started process (PID=2726) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:00.731+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:04:00.735+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:04:00.734+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:00.739+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:04:00.738+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:04:00.740+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:00.779+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.057 seconds
[2025-03-19T23:04:31.025+0000] {processor.py:186} INFO - Started process (PID=2789) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:31.034+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:04:31.038+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:04:31.038+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:31.047+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:04:31.043+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:04:31.048+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:04:31.105+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.091 seconds
[2025-03-19T23:05:01.845+0000] {processor.py:186} INFO - Started process (PID=2851) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:01.849+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:05:01.852+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:05:01.852+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:01.858+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:05:01.856+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:05:01.859+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:01.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.055 seconds
[2025-03-19T23:05:32.379+0000] {processor.py:186} INFO - Started process (PID=2915) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:32.385+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:05:32.389+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:05:32.388+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:32.394+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:05:32.393+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:05:32.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:05:32.498+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.128 seconds
[2025-03-19T23:06:02.643+0000] {processor.py:186} INFO - Started process (PID=2979) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:02.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:06:02.652+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:06:02.651+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:02.665+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:06:02.660+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:06:02.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:02.701+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.068 seconds
[2025-03-19T23:06:32.873+0000] {processor.py:186} INFO - Started process (PID=3043) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:32.882+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:06:32.886+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:06:32.885+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:32.902+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:06:32.900+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:06:32.904+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:06:32.965+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.118 seconds
[2025-03-19T23:07:03.184+0000] {processor.py:186} INFO - Started process (PID=3107) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:03.188+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:07:03.190+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:07:03.190+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:03.196+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:07:03.194+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:07:03.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:03.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.052 seconds
[2025-03-19T23:07:33.405+0000] {processor.py:186} INFO - Started process (PID=3171) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:33.420+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:07:33.424+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:07:33.423+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:33.431+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:07:33.429+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:07:33.431+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:07:33.467+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.073 seconds
[2025-03-19T23:08:03.972+0000] {processor.py:186} INFO - Started process (PID=3234) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:03.982+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:08:03.987+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:08:03.986+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:03.992+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:08:03.991+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:08:03.993+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:04.021+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.062 seconds
[2025-03-19T23:08:34.081+0000] {processor.py:186} INFO - Started process (PID=3298) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:34.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:08:34.094+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:08:34.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:34.103+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:08:34.100+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:08:34.104+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:08:34.135+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.064 seconds
[2025-03-19T23:09:04.415+0000] {processor.py:186} INFO - Started process (PID=3360) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:04.418+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:09:04.422+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:04.421+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:04.428+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:04.426+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 2, in <module>
    from request import Response
ModuleNotFoundError: No module named 'request'
[2025-03-19T23:09:04.429+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:04.474+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.070 seconds
[2025-03-19T23:09:07.073+0000] {processor.py:186} INFO - Started process (PID=3363) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:07.078+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:09:07.084+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:07.083+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:07.106+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:07.104+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:09:07.107+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:07.186+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.132 seconds
[2025-03-19T23:09:37.295+0000] {processor.py:186} INFO - Started process (PID=3428) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:37.298+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:09:37.307+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:37.306+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:37.319+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:09:37.317+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:09:37.320+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:09:37.344+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.073 seconds
[2025-03-19T23:10:07.551+0000] {processor.py:186} INFO - Started process (PID=3492) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:07.555+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:10:07.560+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:10:07.559+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:07.575+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:10:07.573+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:10:07.576+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:07.644+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.107 seconds
[2025-03-19T23:10:37.838+0000] {processor.py:186} INFO - Started process (PID=3552) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:37.846+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:10:37.854+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:10:37.854+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:37.866+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:10:37.863+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:10:37.871+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:10:37.913+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.118 seconds
[2025-03-19T23:11:11.577+0000] {processor.py:186} INFO - Started process (PID=3609) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:11.580+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:11:11.588+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:11:11.587+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:11.610+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:11:11.596+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:11:11.612+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:11.739+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.183 seconds
[2025-03-19T23:11:44.994+0000] {processor.py:186} INFO - Started process (PID=3634) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:45.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:11:45.011+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:11:45.010+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:45.022+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:11:45.019+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:11:45.023+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:11:45.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.128 seconds
[2025-03-19T23:12:18.865+0000] {processor.py:186} INFO - Started process (PID=3696) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:12:18.876+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:12:18.880+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:12:18.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:12:18.894+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:12:18.889+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:12:18.895+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:12:18.988+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.138 seconds
[2025-03-19T23:13:02.163+0000] {processor.py:186} INFO - Started process (PID=3765) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:02.214+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:13:02.259+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:13:02.217+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:02.336+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:13:02.279+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:13:02.338+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:02.768+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.709 seconds
[2025-03-19T23:13:33.371+0000] {processor.py:186} INFO - Started process (PID=3802) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:33.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:13:33.396+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:13:33.395+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:33.412+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:13:33.409+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:13:33.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:13:33.642+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.283 seconds
[2025-03-19T23:14:03.898+0000] {processor.py:186} INFO - Started process (PID=3864) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:14:03.906+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:14:03.909+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:14:03.909+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:14:03.914+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:14:03.913+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:14:03.915+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:14:03.992+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.105 seconds
[2025-03-19T23:18:54.743+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:18:54.746+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:18:54.765+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:18:54.750+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:18:54.774+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:18:54.771+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:18:54.776+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:18:54.854+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.121 seconds
[2025-03-19T23:19:34.541+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:19:34.615+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:19:34.630+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:19:34.619+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:19:34.638+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:19:34.635+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:19:34.639+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:19:34.845+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.343 seconds
[2025-03-19T23:20:18.743+0000] {processor.py:186} INFO - Started process (PID=188) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:18.745+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:20:18.750+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:20:18.749+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:18.761+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:20:18.758+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:20:18.762+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:18.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.104 seconds
[2025-03-19T23:20:49.588+0000] {processor.py:186} INFO - Started process (PID=250) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:49.593+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:20:49.598+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:20:49.597+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:49.608+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:20:49.605+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:20:49.610+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:20:49.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.083 seconds
[2025-03-19T23:21:21.831+0000] {processor.py:186} INFO - Started process (PID=306) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:21.838+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:21:21.842+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:21:21.841+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:21.849+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:21:21.847+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:21:21.851+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:21.983+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.161 seconds
[2025-03-19T23:21:52.156+0000] {processor.py:186} INFO - Started process (PID=369) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:52.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:21:52.160+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:21:52.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:52.164+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:21:52.163+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:21:52.165+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:21:52.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.038 seconds
[2025-03-19T23:22:22.368+0000] {processor.py:186} INFO - Started process (PID=431) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:22.370+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:22:22.374+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:22:22.373+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:22.380+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:22:22.379+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:22:22.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:22.415+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.058 seconds
[2025-03-19T23:22:52.567+0000] {processor.py:186} INFO - Started process (PID=495) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:52.576+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:22:52.581+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:22:52.580+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:52.587+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:22:52.586+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:22:52.588+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:22:52.620+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.063 seconds
[2025-03-19T23:23:22.842+0000] {processor.py:186} INFO - Started process (PID=559) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:22.844+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:23:22.847+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:23:22.847+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:22.853+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:23:22.851+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:23:22.854+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:22.877+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.044 seconds
[2025-03-19T23:23:53.592+0000] {processor.py:186} INFO - Started process (PID=629) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:53.604+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:23:53.608+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:23:53.607+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:53.614+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:23:53.612+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:23:53.614+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:23:53.643+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.064 seconds
[2025-03-19T23:24:24.237+0000] {processor.py:186} INFO - Started process (PID=693) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:24.249+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:24:24.253+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:24:24.253+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:24.258+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:24:24.257+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:24:24.259+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:24.288+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.059 seconds
[2025-03-19T23:24:54.926+0000] {processor.py:186} INFO - Started process (PID=757) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:54.928+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:24:54.930+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:24:54.930+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:54.935+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:24:54.934+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:24:54.936+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:24:54.962+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.047 seconds
[2025-03-19T23:25:25.458+0000] {processor.py:186} INFO - Started process (PID=821) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:25.492+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:25:25.517+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:25:25.516+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:25.531+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:25:25.530+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:25:25.532+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:25.575+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.269 seconds
[2025-03-19T23:25:55.671+0000] {processor.py:186} INFO - Started process (PID=884) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:55.673+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:25:55.677+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:25:55.676+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:55.682+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:25:55.681+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:25:55.683+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:25:55.716+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.054 seconds
[2025-03-19T23:26:25.776+0000] {processor.py:186} INFO - Started process (PID=947) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:25.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:26:25.779+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:26:25.779+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:25.784+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:26:25.783+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:26:25.785+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:25.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.044 seconds
[2025-03-19T23:26:55.972+0000] {processor.py:186} INFO - Started process (PID=1010) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:56.057+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:26:56.061+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:26:56.061+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:56.066+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:26:56.065+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:26:56.067+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:26:56.211+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.249 seconds
[2025-03-19T23:27:26.909+0000] {processor.py:186} INFO - Started process (PID=1071) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:26.922+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:27:26.924+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:27:26.924+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:26.930+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:27:26.928+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:27:26.931+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:26.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.059 seconds
[2025-03-19T23:27:57.666+0000] {processor.py:186} INFO - Started process (PID=1135) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:57.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:27:57.680+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:27:57.679+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:57.688+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:27:57.685+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:27:57.689+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:27:57.724+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.074 seconds
[2025-03-19T23:28:28.348+0000] {processor.py:186} INFO - Started process (PID=1198) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:28.350+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:28:28.353+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:28:28.353+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:28.358+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:28:28.357+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:28:28.358+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:28.385+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T23:28:58.901+0000] {processor.py:186} INFO - Started process (PID=1262) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:58.908+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:28:58.912+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:28:58.911+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:58.919+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:28:58.917+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:28:58.919+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:28:58.946+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.059 seconds
[2025-03-19T23:29:29.559+0000] {processor.py:186} INFO - Started process (PID=1326) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:29:29.560+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:29:29.564+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:29:29.563+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:29:29.569+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:29:29.568+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:29:29.570+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:29:29.600+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.050 seconds
[2025-03-19T23:30:00.100+0000] {processor.py:186} INFO - Started process (PID=1390) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:00.104+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:30:00.108+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:30:00.107+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:00.113+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:30:00.112+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:30:00.114+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:00.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T23:30:30.765+0000] {processor.py:186} INFO - Started process (PID=1455) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:30.772+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:30:30.774+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:30:30.774+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:30.779+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:30:30.778+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:30:30.780+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:30:30.804+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.046 seconds
[2025-03-19T23:31:01.071+0000] {processor.py:186} INFO - Started process (PID=1519) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:01.073+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:31:01.076+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:31:01.076+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:01.080+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:31:01.079+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:31:01.081+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:01.129+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.070 seconds
[2025-03-19T23:31:31.884+0000] {processor.py:186} INFO - Started process (PID=1583) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:31.891+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:31:31.895+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:31:31.894+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:31.900+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:31:31.899+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:31:31.901+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:31:31.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T23:32:02.679+0000] {processor.py:186} INFO - Started process (PID=1646) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:02.681+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:32:02.685+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:32:02.685+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:02.691+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:32:02.688+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:32:02.693+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:02.725+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.056 seconds
[2025-03-19T23:32:33.257+0000] {processor.py:186} INFO - Started process (PID=1710) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:33.268+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:32:33.272+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:32:33.272+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:33.277+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:32:33.276+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:32:33.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:32:33.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.058 seconds
[2025-03-19T23:33:03.665+0000] {processor.py:186} INFO - Started process (PID=1774) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:03.673+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:33:03.676+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:33:03.675+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:03.679+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:33:03.678+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:33:03.680+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:03.702+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.048 seconds
[2025-03-19T23:33:33.883+0000] {processor.py:186} INFO - Started process (PID=1838) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:33.886+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:33:33.890+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:33:33.890+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:33.897+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:33:33.895+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:33:33.898+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:33:33.924+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.054 seconds
[2025-03-19T23:34:03.963+0000] {processor.py:186} INFO - Started process (PID=1902) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:03.967+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:34:03.972+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:34:03.971+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:03.976+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:34:03.975+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:34:03.976+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:04.009+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.052 seconds
[2025-03-19T23:34:34.966+0000] {processor.py:186} INFO - Started process (PID=1957) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:34.968+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:34:34.970+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:34:34.970+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:34.975+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:34:34.974+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:34:34.975+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:34:35.001+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.043 seconds
[2025-03-19T23:35:05.687+0000] {processor.py:186} INFO - Started process (PID=2018) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:05.689+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:35:05.692+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:35:05.691+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:05.696+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:35:05.695+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:35:05.696+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:05.721+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.047 seconds
[2025-03-19T23:35:36.325+0000] {processor.py:186} INFO - Started process (PID=2083) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:36.327+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:35:36.331+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:35:36.330+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:36.336+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:35:36.334+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:35:36.337+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:35:36.366+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.050 seconds
[2025-03-19T23:36:06.865+0000] {processor.py:186} INFO - Started process (PID=2147) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:06.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:36:06.880+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:36:06.880+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:06.886+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:36:06.884+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:36:06.887+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:06.921+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.068 seconds
[2025-03-19T23:36:37.297+0000] {processor.py:186} INFO - Started process (PID=2211) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:37.313+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:36:37.316+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:36:37.316+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:37.320+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:36:37.319+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:36:37.321+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:36:37.351+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.063 seconds
[2025-03-19T23:37:07.815+0000] {processor.py:186} INFO - Started process (PID=2275) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:07.822+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:37:07.826+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:37:07.825+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:07.833+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:37:07.831+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:37:07.834+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:07.861+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.057 seconds
[2025-03-19T23:37:38.222+0000] {processor.py:186} INFO - Started process (PID=2338) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:38.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:37:38.232+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:37:38.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:38.236+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:37:38.235+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:37:38.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:37:38.262+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T23:38:08.489+0000] {processor.py:186} INFO - Started process (PID=2402) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:08.499+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:38:08.503+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:38:08.503+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:08.510+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:38:08.508+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:38:08.511+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:08.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.104 seconds
[2025-03-19T23:38:38.861+0000] {processor.py:186} INFO - Started process (PID=2464) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:38.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:38:38.869+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:38:38.869+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:38.876+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:38:38.873+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:38:38.877+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:38:38.945+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.093 seconds
[2025-03-19T23:39:09.091+0000] {processor.py:186} INFO - Started process (PID=2519) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:09.093+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:39:09.095+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:39:09.095+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:09.099+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:39:09.098+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:39:09.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:09.125+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.044 seconds
[2025-03-19T23:39:39.708+0000] {processor.py:186} INFO - Started process (PID=2579) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:39.710+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:39:39.713+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:39:39.713+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:39.717+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:39:39.716+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:39:39.718+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:39:39.742+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.040 seconds
[2025-03-19T23:40:09.829+0000] {processor.py:186} INFO - Started process (PID=2639) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:09.831+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:40:09.834+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:40:09.834+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:09.840+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:40:09.839+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:40:09.841+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:09.886+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.068 seconds
[2025-03-19T23:40:40.326+0000] {processor.py:186} INFO - Started process (PID=2703) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:40.333+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:40:40.336+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:40:40.335+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:40.340+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:40:40.339+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:40:40.340+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:40:40.363+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.050 seconds
[2025-03-19T23:41:10.644+0000] {processor.py:186} INFO - Started process (PID=2767) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:10.645+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:41:10.648+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:41:10.647+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:10.653+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:41:10.652+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:41:10.654+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:10.678+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.040 seconds
[2025-03-19T23:41:40.826+0000] {processor.py:186} INFO - Started process (PID=2829) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:40.834+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:41:40.837+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:41:40.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:40.841+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:41:40.840+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:41:40.842+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:41:40.871+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.055 seconds
[2025-03-19T23:42:11.833+0000] {processor.py:186} INFO - Started process (PID=2893) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:11.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:42:11.843+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:42:11.843+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:11.849+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:42:11.848+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:42:11.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:11.878+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.056 seconds
[2025-03-19T23:42:42.791+0000] {processor.py:186} INFO - Started process (PID=2956) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:42.793+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:42:42.797+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:42:42.796+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:42.803+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:42:42.801+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:42:42.803+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:42:42.835+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.055 seconds
[2025-03-19T23:43:13.025+0000] {processor.py:186} INFO - Started process (PID=3020) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:13.037+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:43:13.039+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:43:13.039+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:13.043+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:43:13.042+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:43:13.044+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:13.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.057 seconds
[2025-03-19T23:43:43.764+0000] {processor.py:186} INFO - Started process (PID=3084) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:43.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:43:43.775+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:43:43.775+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:43.781+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:43:43.779+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:43:43.782+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:43:43.810+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.060 seconds
[2025-03-19T23:44:14.285+0000] {processor.py:186} INFO - Started process (PID=3148) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:14.304+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:44:14.307+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:44:14.307+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:14.314+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:44:14.312+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:44:14.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:14.346+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.081 seconds
[2025-03-19T23:44:44.968+0000] {processor.py:186} INFO - Started process (PID=3212) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:44.975+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:44:44.980+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:44:44.980+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:44.990+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:44:44.986+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:44:44.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:44:45.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.062 seconds
[2025-03-19T23:45:15.415+0000] {processor.py:186} INFO - Started process (PID=3276) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:15.427+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:45:15.430+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:45:15.429+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:15.434+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:45:15.433+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:45:15.435+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:15.461+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.054 seconds
[2025-03-19T23:45:45.799+0000] {processor.py:186} INFO - Started process (PID=3339) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:45.802+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:45:45.806+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:45:45.806+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:45.813+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:45:45.811+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:45:45.814+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:45:45.838+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.051 seconds
[2025-03-19T23:46:16.616+0000] {processor.py:186} INFO - Started process (PID=3403) to work on /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:46:16.634+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bronze_layer.py for tasks to queue
[2025-03-19T23:46:16.638+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:46:16.637+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:46:16.644+0000] {logging_mixin.py:190} INFO - [2025-03-19T23:46:16.643+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bronze_layer.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bronze_layer.py", line 6, in <module>
    from pyspark.sql import SparkSession
ModuleNotFoundError: No module named 'pyspark'
[2025-03-19T23:46:16.656+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bronze_layer.py
[2025-03-19T23:46:16.742+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bronze_layer.py took 0.140 seconds
